lib/modules.nix

```diff
@@ -708,6 +715,23 @@ rec {
     else
       [ def ];
 
+  # Apply a list of functions starting from the left.
+  # High priority functions get applied first (mkBefore), low priority functions get applied last (mkAfter) and return the final result.
+  applyAll = functions: x:
+    if functions != [] then
+      applyAll (tail functions) ((head functions) x)
+    else
+      x;
```

Comment:
that's just a really complicated (and inefficient) way to write `foldl (a: f: f a)`, right?

Reply:
I thought it couldn't be done with fold but I guess I was wrong